build/images/scripts/start_ovs_windows_containerd.ps1

```diff
@@ -0,0 +1,166 @@
+       $PSModuleFiles | ForEach-Object {
+           $PSModulePath = Join-Path -Path $PowerShellModuleBase -ChildPath $_.BaseName
+           if (!(Test-Path $PSModulePath)) {
+               Log "Installing $_"
+               mkdir -p $PSModulePath
+               Copy-Item $_.FullName $PSModulePath
+           }
+       }
+   }
+
+   # Install OVS kernel driver.
+   Log "Installing OVS kernel driver"
+   $VMMSStatus = $(Get-Service vmms -ErrorAction SilentlyContinue).Status
+   if (!$VMMSStatus) {
+       $VMMSStatus = "not exist"
+   }
+   Log "Hyper-V Virtual Machine Management service status: $VMMSStatus"
+   if ($VMMSStatus -eq "Running") {
+       cmd /c "cd $OVSDriverDir && install.cmd"
+   } else {
+       cd $OVSDriverDir ; netcfg -l .\ovsext.inf -c s -i OVSExt; cd $WorkDir
+   }
+   if (!$?) {
+       Log "Install OVS kernel driver failed, exit"
+       exit 1
+   }
+}
+
+function ConfigOVS() {
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
+    $env:PATH = $env:PATH + ";$mountPath/Windows/System32;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
+    $OVS_DB_SCHEMA_PATH = "$mountPath/openvswitch/usr/share/openvswitch/vswitch.ovsschema"
+    $OVS_DB_PATH = "C:\openvswitch\etc\openvswitch\conf.db"
+    if ($(Test-Path $OVS_DB_SCHEMA_PATH) -and !$(Test-Path $OVS_DB_PATH)) {
+        ovsdb-tool create "$OVS_DB_PATH" "$OVS_DB_SCHEMA_PATH"
+    }
+    ovsdb-server $OVS_DB_PATH -vfile:info --remote=punix:db.sock --log-file=/var/log/antrea/openvswitch/ovsdb-server.log --pidfile --detach
+    ovs-vsctl --no-wait init
+
+    # Set OVS version.
+    $OVS_VERSION=$(Get-Item $mountPath\openvswitch\driver\OVSExt.sys).VersionInfo.ProductVersion
+    ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION
+
+    ovs-vswitchd --log-file=/var/log/antrea/openvswitch/ovs-vswitchd.log --pidfile -vfile:info --detach
+
+    $SleepInterval = 30
+    Write-Host "Started the loop that checks OVS status every $SleepInterval seconds"
+    while ($true) {
+        if ( !( Get-Process ovsdb-server ) ) {
+            Write-Host "ovsdb-server is not running, starting it again..."
+            ovsdb-server $OVS_DB_PATH -vfile:info --remote=punix:db.sock --log-file=/var/log/antrea/openvswitch/ovsdb-server.log --pidfile --detach
+        }
+        if ( !( Get-Process ovs-vswitchd ) ) {
+            Write-Host "ovs-vswitchd is not running, starting it again..."
+            ovs-vswitchd --log-file=/var/log/antrea/openvswitch/ovs-vswitchd.log --pidfile -vfile:info --detach
+        }
+        Start-Sleep -Seconds $SleepInterval
+    }
+}
+
+Log "Installation log location: $InstallLog"
+
+CheckIfOVSInstalled
+
+DownloadOVS
+
+InstallOVS
+
+# Antrea Pod runs as NT AUTHORITY\SYSTEM user on Windows, antrea-ovs container writes
+# pid and conf.db files to $OVSInstallDir on Windows host Node during runtime.
+icacls $OVSInstallDir /grant "NT AUTHORITY\SYSTEM:(OI)(CI)F" /T
```

Comment:
Is it required here? 
Previously this command was run by administrator before running antrea as antrea Pod runs as NT AUTHORITY\SYSTEM user on Windows. Here, this cmd is executed after running the antrea Pod as this script is run by antrea Pod.

Reply:
removed